Samples/excel-create-worksheet-from-web-site/assets/sample.json

```diff
@@ -0,0 +1,46 @@
+[
+    {
+      "name": "excel-add-in-create-spreadsheet-from-web-page",
```

Comment:
Should this say "page" or "site"?

Reply:
The intent is about your web site (any page on your site, or perhaps even a shared ribbon could be the source to create a spreadsheet.

Reply:
oh the article is inconsistent. I need to reexamine the usage here and make them consistent.

Reply:
Page is fine. changed here.